2932 docs rfc rewrite the get started and compare with get started in console

billy-the-fish opened the pull request:
# Description

[Short summary of why you created this PR]

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

billy-the-fish committed:
First go at the get started rewrite

billy-the-fish committed:
Merge branch 'rewrite-get-started' of github.com:billy-the-fish/timescale-docs into 2932-docs-rfc-rewrite-the-get-started-and-compare-with-get-started-in-console

jamessewell committed:
Update index.md

Signed-off-by: James Sewell <james.sewell@gmail.com>

jamessewell commented on getting-started/index.md:
I would love to get the concept of usage-based storage into the Postgres section (you don't choose a fixed storage amount you pay for what you use) because that's a big differentiator.

jamessewell replied:
Oh and a mention of Vector as well? I'm very conscious that people enter our ecosystem on this page :)

billy-the-fish replied:
Nice point. Updated, I think it looks a lot better.

jamessewell reviewed (commented):
Looks good -> see usage based storage comment

jamessewell committed:
Merge branch 'latest' into 2932-docs-rfc-rewrite-the-get-started-and-compare-with-get-started-in-console

jamessewell committed:
Update services.md

Signed-off-by: James Sewell <james.sewell@gmail.com>

jamessewell committed:
Update index.md

Signed-off-by: James Sewell <james.sewell@gmail.com>

billy-the-fish committed:
feat: rewrite the get started.

billy-the-fish committed:
Merge branch 'latest' into 2932-docs-rfc-rewrite-the-get-started-and-compare-with-get-started-in-console

billy-the-fish committed:
feat: remove extra doc and update redirects.

billy-the-fish commented:
Updated the intro in https://docs-dev.timescale.com/docs-2932-docs-rfc-rewrite-the-get-started-and-compare-with-get-started-in-console/getting-started/2932-docs-rfc-rewrite-the-get-started-and-compare-with-get-started-in-console/ to mention paying for data. Have removed https://docs.timescale.com/use-timescale/latest/services/create-a-service/ from the docs and updated the redirects file to point to the new get started.

jamessewell committed:
Update the services page

jamessewell committed:
Demanding -> resource-intensive workloads

jamessewell committed:
Merge branch 'latest' into 2932-docs-rfc-rewrite-the-get-started-and-compare-with-get-started-in-console

billy-the-fish committed:
feat: last language update.
